bin/ci

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash --login
```

Comment:
What do you think of using the more portable `#!/usr/bin/env bash` shebang form?